src/components/Person/Person.spec.jsx

```diff
@@ -51,21 +48,16 @@ describe('Person', () => {
         isMarried: false,
         partnerName: 'Some Partner',
       };
-
       mount(<Person person={tanya} />);
-
       cy.get('.Person__partner').should('contain.text', 'not married');
     });
-
     it('should not show age if age is omitted', () => {
       const sasha = {
         name: 'Sasha',
         sex: 'f',
         isMarried: false,
       };
-
       mount(<Person person={sasha} />);
-
       cy.get('.Person__age').should('not.exist');
```

Comment:
The test case checks that the `.Person__age` element does not exist when age is omitted, which is correct. However, it might be beneficial to also check that the greeting message displays 'unknown' for the age in this scenario, as per the component's implementation.